account_invoice_tax_note/reports/report_invoice_document.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2018 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <template id="report_invoice_document_inherit" inherit_id="account.report_invoice_document">
+        <p t-if="o.fiscal_position_id.note" position="after">
+            <t t-set="account_tax_groups" t-value="o._get_account_tax_groups()"/>
+            <t t-if="account_tax_groups">
+                <t t-foreach="account_tax_groups" t-as="account_tax_group">
+                    <p t-field="account_tax_group.report_note"/>
+                </t>
```

Comment:
skip empty notes?

Reply:
Fixed (into _get_account_tax_groups() function of account.invoice)